crates/augurs-forecaster/src/transforms.rs

```diff
@@ -81,7 +90,15 @@ impl Transform {
         Self::Log
     }
 
-    pub(crate) fn transform<'a, T>(&'a self, input: T) -> Box<dyn Iterator<Item = f64> + 'a>
+    /// Create a new Box-Cox transform.
+    ///
+    /// This transform applies the Box-Cox transformation to each item.
+    pub fn boxcox(lambda: f64) -> Self {
+        Self::BoxCox(lambda)
+    }
+
+    /// Apply the transformation to the given time series.
+    pub fn transform<'a, T>(&'a self, input: T) -> Box<dyn Iterator<Item = f64> + 'a>
```

Comment:
not sure if pub is required here either - would rather keep as many things as possible hidden.